curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f356ed63e0fa262326eef05.md

```diff
@@ -7,11 +7,11 @@ dashedName: step-24
 
 # --description--
 
-Now make the background color of the `div` element to be `burlywood`.
+Now use the existing `div` selector to set the background color of the `div` element to be `burlywood`.
 
 # --hints--
 
-You should set the `background-color` property to `burlywood`.
+In the existing `div` selector, change the background color to `burlywood`.
```

Comment:
I think we should still modify this slightly. It isn't really a hint if it says basically the same thing as the instructions. I think I would just revert this to what it was originally...

```suggestion
You should set the `background-color` property to `burlywood`.
```

Keeping the `background-color` as a code block lets users know that that is the specific property they need to use. Also, the test doesn't check that the div has that value - it just checks that something has that value. So the idea would be that this hint lets them know what property and value to use, and the next hint would let them know to use it in the div.

Reply:
Yes, the code block for the property name results in a better hint. Thank you for your input!